sql3/planner/inbuiltfunctionsstring.go

```diff
@@ -644,3 +660,34 @@ func (n *callPlanExpression) EvaluateReplicate(currentRow []interface{}) (interf
 	return strings.Repeat(stringArg, int(intArg)), nil
 
 }
+
+// implements fmt.Sprintf
```

Comment:
Just for reference, exported methods (and really any function, type, or method which is complex enough to benefit from having a description) should have godocs formatted like:
```
// EvalutateFormat ... [and then describe what it does and any information that would be useful for a user or another developer to know]
```
More info about that here: https://go.dev/blog/godoc